src/core/infrastructure/event.listener.js

```diff
@@ -5,12 +5,12 @@ export class EventListener {
 		this.handlers = {};
 	}
 
-	on(name, f) {
+	on(name, f, settings = null) {
```

Comment:
Can we set settings=false?

Reply:
or/and change variable name, as I understand it's just boolean ```useCapture``` argument of ```addEventListener```. Does it make sense?

Reply:
I mean default parameter applies when value is not passed, we set it into null and after if null we use false. Why?  We can just set default to false.

Reply:
@titov-max that's a good point. Old standards defined third parameter as boolean `useCapture`. But now it can be [a hash](https://dom.spec.whatwg.org/#interface-eventtarget) . I think this should be noticed, because it may not work with  <51 Chrome versions

Reply:
Oh, right, I see. Thank you, folks. A hash confuses a bit really.

Reply:
@vkorolev, a good point about old browsers support